### ISSUE TITLE
Clicking on error code link in Visual Basic windows causesbrowser to open invalid link

### ISSUE THREAD

**Issue by paul1956** [dotnet/roslyn#5003]
Clicking on CC0014 (or other errors) caused Microsoft Edge browser to open link below which returns 404 for all the errors I tested.
```
https://code-cracker.github.io/diagnostics/CC0014.html
```

**Comment by Pilchie** [comment]
This should be reported at https://github.com/code-cracker/code-cracker, since that package provides that error.

**Comment by paul1956** [comment]
Thanks
